addOns/requester/src/main/resources/org/zaproxy/addon/requester/Messages.properties

```diff
@@ -15,6 +15,7 @@ requester.httppanel.find.find.match.1 = 1 of 1 match
 requester.httppanel.find.find.match.x = {0} of {1} matches
 requester.httppanel.find.next = Next match
 requester.httppanel.find.prev = Previous match
+requester.httppanel.ioexception.exception = IO error in sending request: \n{0}: {1}
```

Comment:
The space before the newlines should be removed. I'm not sure that showing the class name helps much the user.